audio/audacious-plugins/Portfile

```diff
@@ -128,6 +128,17 @@ configure.args-append \
 configure.args-append \
                     -Dmoonstone=false
 
+# This issue was supposed to be fixed, but it did not work as intended.
+# Temporarily revert upstream commit and restore our patch. This is tested.
+# https://github.com/audacious-media-player/audacious-plugins/pull/166
+patchfiles-append   0001-xsf-desmume-types.h-ensure-LOCAL_BE-gets-defined-on-.patch \
+                    patch-big-endian.diff
+
+# SPInvocationGrabbing.m: error: '-fobjc-exceptions' is required to enable Objective-C exception syntax
+# SPInvocationGrabbing.m: error: 'for' loop initial declarations are only allowed in C99 or C11 mode
+# SPInvocationGrabbing.m: note: use option '-std=c99', '-std=gnu99', '-std=c11' or '-std=gnu11' to compile your code
+# patchfiles-append   0002-mac-media-keys-use-correct-objc-flags.patch
```

Comment:
What are these lines for?

Reply:
Probably an atavism. I will remove these.